pla/en/helper-loader.texy

```diff
@@ -112,17 +114,21 @@ common:
 \--
 
 
-Than we can register `Helper loader` in BasePresenter.php by callback to our method loader() into Template::registerHelperLoader().
+Than we can register `Filter loader` in BasePresenter.php by callback to our method loader() into Template::addFilter().
 
 /--php
 abstract class BasePresenter extends Nette\Application\UI\Presenter
 {
+    /**
+    * @var \App\Helpers @inject
```

Comment:
This classname might be confusing, do you think you could change that to something like `App\TemplateFilters` or `App\RuntimeFilters` ?